### ISSUE TITLE
use pre-trained embedding vectors as entity and relation vectors. However, I've noticed that the computed loss is almost the same as the loss obtained when you originally used randomly generated embeddings with the fix_seed. In most cases

### ISSUE THREAD

**Issue by ShaoZhangHao** [Peyman-Bi/SAttLE#2]
I use pre-trained embedding vectors as entity and relation vectors. However, I've noticed that the computed loss is almost the same as the loss obtained when you originally used randomly generated embeddings with the fix_seed. In most cases, they are exactly the same, even when I removed the fix_seed and allowed all linear layers and initialized embeddings to be random. Why is this happening? What does your code use?
When I use the pretrained embeddin，the first epoch loss is 0.06489 . When I use the randomly generated embedding, the first epoch loss is 0.06489,too.

**Comment by pbaghershahi** [comment]
I'm not sure if I understand you correctly. We don't use any pre-trained model. But if you want to use your own pre-trained parameters for the embedding matrices, have you loaded your parameters after ```self.init_parameters()``` in SAttLE class initialization or commented this line? please make sure this is the case.

**Comment by pbaghershahi** [comment]
> I use pre-trained embedding vectors as entity and relation vectors. However, I've noticed that the computed loss is almost the same as the loss obtained when you originally used randomly generated embeddings with the fix_seed. In most cases, they are exactly the same, even when I removed the fix_seed and allowed all linear layers and initialized embeddings to be random. Why is this happening? What does your code use? When I use the pretrained embeddin，the first epoch loss is 0.06489 . When I use the randomly generated embedding, the first epoch loss is 0.06489,too.

**Comment by pbaghershahi** [comment]
> I use pre-trained embedding vectors as entity and relation vectors. However, I've noticed that the computed loss is almost the same as the loss obtained when you originally used randomly generated embeddings with the fix_seed. In most cases, they are exactly the same, even when I removed the fix_seed and allowed all linear layers and initialized embeddings to be random. Why is this happening? What does your code use? When I use the pretrained embeddin，the first epoch loss is 0.06489 . When I use the randomly generated embedding, the first epoch loss is 0.06489,too.

I'm not sure if I understand you correctly. We don't use any pre-trained model. But if you want to use your own pre-trained parameters for the embedding matrices, have you loaded your parameters after self.init_parameters() in SAttLE class initialization or commented this line? please make sure this is the case.
